site/security.html

```diff
@@ -267,11 +270,14 @@ <h3 id="CVE-2017-7678">CVE-2017-7678 Apache Spark XSS web UI MHTML vulnerability
 PGh0bWw+PHNjcmlwdD5hbGVydCgiWFNTIik8L3NjcmlwdD48L2h0bWw+
 &lt;/div&gt;
 </code></pre>
+
 <p>Result: In the above payload the BASE64 data decodes as:</p>
+
 <pre><code>&lt;html&gt;&lt;script&gt;alert("XSS")&lt;/script&gt;&lt;/html&gt;
 </code></pre>
 
 <p>Credit:</p>
+
```

Comment:
```diff
<h3 id="CVE-2017-7678">CVE-2017-7678 Apache Spark XSS web UI MHTML vulnerability</h3>
"security.md" 95L, 3051C

-<p>Result: In the above payload the BASE64 data decodes as:</p>
-<pre><code>&lt;html&gt;&lt;script&gt;alert("XSS")&lt;/script&gt;&lt;/html&gt;
+</div>
+<pre><code>Result: In the above payload the BASE64 data decodes as:
 </code></pre>
+<html><script>alert("XSS")</script></html>
+<p>```</p>

-<p>Credit:</p>
-<ul>
-  <li>Mike Kasper, Nicholas Marion</li>
-  <li>IBM z Systems Center for Secure Engineering</li>
-</ul>
+<p>Credit:
+- Mike Kasper, Nicholas Marion
+- IBM z Systems Center for Secure Engineering</p>

   </div>
 </div>

```